### PR TITLE
Prepare 1.0.1 release

### PR DESCRIPTION
**Pull request by gzussa**
### What does this PR do?

Prepare changelog for new release.

### Description of the Change

see above

### Verification Process

Based on  https://github.com/jenkinsci/datadog-plugin/compare/datadog-1.0.0...master

### Review checklist (to be filled by reviewers)

- [ ] Feature or bug fix MUST have appropriate tests (unit, integration, etc...)
- [ ] PR title must be written as a CHANGELOG entry [(see why)](/CONTRIBUTING.md#pull-request-title)
- [ ] Files changes must correspond to the primary purpose of the PR as described in the title (small unrelated changes should have their own PR)
- [ ] PR must have one `changelog/` label attached. If applicable it should have the `backward-incompatible` label attached.
- [ ] PR should not have `do-not-merge/` label attached.
- [ ] If Applicable, issue must have `kind/` and `severity/` labels attached at least.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -233,19 +233,25 @@ Build status `jenkins.job.status` with the default tags: : `job`, `node`, `branc
 
 ### Overview
 
-The [DataDog/jenkins-datadog-plugin][8] repository handles the most up-to-date changes made to the Datadog Plugin, as well as issue tickets revolving around that work. Releases are merged to the [Jenkins-CI git repo for our plugin][9], and represents the source used for plugin releases found in the [Update Center][3] in your Jenkins installation.
+The [jenkinsci/datadog-plugin][8] repository handles the most up-to-date changes made to the Datadog Plugin, as well as issue tickets revolving around that work. 
+Releases are merged to the [Jenkins-CI git repo for our plugin][9], and represents the source used for plugin releases found in the [Update Center][3] in your Jenkins installation.
 
-Every commit to the [DataDog/jenkins-datadog-plugin][8] repository triggers a Jenkins build on our internal Jenkins installation.
+Every commit to the repository triggers the Jenkins Org CI pipeline defined in the `JenkinsFile` at the root folder of the source code.
```
Comment (jirikuncar) [inline README.md]:
```suggestion
Every commit to the repository triggers the Jenkins Org CI pipeline defined in the `Jenkinsfile` at the root folder of the source code.
```

File: README.md [hunk 2]
```diff
@@ -233,19 +231,25 @@ Build status `jenkins.job.status` with the default tags: : `job`, `node`, `branc
 
 ### Overview
 
-The [DataDog/jenkins-datadog-plugin][8] repository handles the most up-to-date changes made to the Datadog Plugin, as well as issue tickets revolving around that work. Releases are merged to the [Jenkins-CI git repo for our plugin][9], and represents the source used for plugin releases found in the [Update Center][3] in your Jenkins installation.
+The [jenkinsci/datadog-plugin][8] repository handles the most up-to-date changes made to the Datadog Plugin, as well as issue tickets revolving around that work. 
+Releases are merged to the [Jenkins-CI git repo for our plugin][9], and represents the source used for plugin releases found in the [Update Center][3] in your Jenkins installation.
```
Comment (l0k0ms) [inline README.md]:
```suggestion
Releases are merged to the [Jenkins-CI git repository for the Datadog-plugin][9], and represents the source used for plugin releases found in the [Update Center][3] in your Jenkins installation.
```

File: README.md [hunk 3]
```diff
@@ -233,19 +231,25 @@ Build status `jenkins.job.status` with the default tags: : `job`, `node`, `branc
...
-Every commit to the [DataDog/jenkins-datadog-plugin][8] repository triggers a Jenkins build on our internal Jenkins installation.
+Every commit to the repository triggers the Jenkins Org CI pipeline defined in the `Jenkinsfile` at the root folder of the source code.
 
 A list of releases is available at [jenkinsci/datadog-plugin/releases][10].
 
 ### How to release
 
-To release a new plugin version, change the project version in the [pom.xml][11] from `x.x.x-SNAPSHOT` to the updated version number you would like to see. Add an entry for the new release number to the [CHANGELOG.md][12] file, and ensure that all the changes are listed accurately. Then run the `jenkins-datadog-plugin-release` job in the Jenkins installation. If the job completes successfully, then the newly updated plugin should be available from the Jenkins [Update Center][3] within ~4 hours (plus mirror propogation time).
+To release a new plugin version, change the project version in the [pom.xml][11] from `x.x.x-SNAPSHOT` to the updated version number you would like to see. 
+Add an entry for the new release number to the [CHANGELOG.md][12] file, and ensure that all the changes are listed accurately.
```
Comment (l0k0ms) [inline README.md]:
```suggestion
Add an entry for the new release number to the [CHANGELOG.md][12] file, and ensure that all changes are listed accurately. 
```

File: README.md [hunk 4]
```diff
@@ -233,19 +231,25 @@ Build status `jenkins.job.status` with the default tags: : `job`, `node`, `branc
...
-The [DataDog/jenkins-datadog-plugin][8] repository handles the most up-to-date changes made to the Datadog Plugin, as well as issue tickets revolving around that work. Releases are merged to the [Jenkins-CI git repo for our plugin][9], and represents the source used for plugin releases found in the [Update Center][3] in your Jenkins installation.
+The [jenkinsci/datadog-plugin][8] repository handles the most up-to-date changes made to the Datadog Plugin, as well as issue tickets revolving around that work. 
+Releases are merged to the [Jenkins-CI git repo for our plugin][9], and represents the source used for plugin releases found in the [Update Center][3] in your Jenkins installation.
 
-Every commit to the [DataDog/jenkins-datadog-plugin][8] repository triggers a Jenkins build on our internal Jenkins installation.
+Every commit to the repository triggers the Jenkins Org CI pipeline defined in the `Jenkinsfile` at the root folder of the source code.
 
 A list of releases is available at [jenkinsci/datadog-plugin/releases][10].
 
 ### How to release
```
Comment (l0k0ms) [inline README.md]: I think we should do a bullet list here:

```
To release a new Datadog-plugin version:

1. Change the project version in the [pom.xml][11] from `x.x.x-SNAPSHOT` to the updated version number you would like to see. 
2. Add an entry for the new release number to the [CHANGELOG.md][12] file, and ensure that all changes are listed accurately. 
3.  Clone to repository and checkout the `master` branch.
4. For repo admins only - Trigger the release by running the following command `mvn -DlocalCheckout=true release:prepare release:perform`.
If the command completes successfully, then the newly updated plugin should be available from the Jenkins [Update Center][3] within ~4 hours (plus mirror propagation time).

File: README.md [hunk 5]
```diff
@@ -233,19 +231,27 @@ Build status `jenkins.job.status` with the default tags: : `job`, `node`, `branc
...
+4. For repository admins only - Trigger the release by running the following command `mvn -DlocalCheckout=true release:prepare release:perform`.
+If the command completes successfully, then the newly updated plugin should be available from the Jenkins [Update Center][3] within ~4 hours (plus mirror propagation time).
 
 ## Issue Tracking
 
-Github's built-in issue tracking system is used to track all issues relating to this plugin: [DataDog/jenkins-datadog-plugin/issues][13]. However, given how Jenkins plugins are hosted, there may be issues that are posted to JIRA as well. You can check [this jenkins issue][14] for those issue postings.
+Github's built-in issue tracking system is used to track all issues relating to this plugin: [jenkinsci/datadog-plugin/issues][13]. 
+However, given how Jenkins plugins are hosted, there may be issues that are posted to JIRA as well. You can check [this jenkins issue][14] for those issue postings.
 
 **Note**: [Unresolved issues on JIRA mentioning Datadog.][15].
```
Comment (l0k0ms) [inline README.md]:
```suggestion
**Note**: [Unresolved issues on JIRA mentioning Datadog][15].
```

File: README.md [hunk 6]
```diff
@@ -233,19 +231,27 @@ Build status `jenkins.job.status` with the default tags: : `job`, `node`, `branc
...
+1. Change the project version in the [pom.xml][11] from `x.x.x-SNAPSHOT` to the updated version number you would like to see. 
+2. Add an entry for the new release number to the [CHANGELOG.md][12] file, and ensure that all changes are listed accurately. 
+3. Clone to repository and checkout the `master` branch.
+4. For repository admins only - Trigger the release by running the following command `mvn -DlocalCheckout=true release:prepare release:perform`.
+If the command completes successfully, then the newly updated plugin should be available from the Jenkins [Update Center][3] within ~4 hours (plus mirror propagation time).
 
 ## Issue Tracking
 
-Github's built-in issue tracking system is used to track all issues relating to this plugin: [DataDog/jenkins-datadog-plugin/issues][13]. However, given how Jenkins plugins are hosted, there may be issues that are posted to JIRA as well. You can check [this jenkins issue][14] for those issue postings.
+Github's built-in issue tracking system is used to track all issues relating to this plugin: [jenkinsci/datadog-plugin/issues][13].
```
Comment (jirikuncar) [inline README.md]: `s/Github/GitHub/`

File: README.md [hunk 7]
```diff
@@ -233,19 +231,27 @@ Build status `jenkins.job.status` with the default tags: : `job`, `node`, `branc
...
 
 ### How to release
 
-To release a new plugin version, change the project version in the [pom.xml][11] from `x.x.x-SNAPSHOT` to the updated version number you would like to see. Add an entry for the new release number to the [CHANGELOG.md][12] file, and ensure that all the changes are listed accurately. Then run the `jenkins-datadog-plugin-release` job in the Jenkins installation. If the job completes successfully, then the newly updated plugin should be available from the Jenkins [Update Center][3] within ~4 hours (plus mirror propogation time).
+To release a new Datadog-plugin version:
+
+1. Change the project version in the [pom.xml][11] from `x.x.x-SNAPSHOT` to the updated version number you would like to see. 
+2. Add an entry for the new release number to the [CHANGELOG.md][12] file, and ensure that all changes are listed accurately. 
+3. Clone to repository and checkout the `master` branch.
+4. For repository admins only - Trigger the release by running the following command `mvn -DlocalCheckout=true release:prepare release:perform`.
```
Comment (jirikuncar) [inline README.md]:
```suggestion
4. For repository admins only - trigger the release by running the following command `mvn -DlocalCheckout=true release:prepare release:perform`.
```

**Pull request merged**